packages/haste-task-webpack/package.json

```diff
@@ -8,6 +8,7 @@
   },
   "devDependencies": {
     "haste-test-utils": "^0.1.13",
+    "retry-promise": "^1.0.0",
```

Comment:
I would use fixed version for [this package](https://github.com/olalonde/retry-promise/releases)